lib/modules/datasource/custom/readme.md

```diff
@@ -1,16 +1,15 @@
-This `custom` datasource allows requesting version data from generic HTTP endpoints.
+This `custom` datasource allows requesting version data from generic HTTP(S) endpoints.
 
 ## Usage
 
 The `customDatasources` option takes a record of `customDatasource` configs.
-This example shows how to update the `k3s.version` file with a custom datasource and
-a [regexManagers](../../manager/regex/):
+This example shows how to update the `k3s.version` file with a custom datasource and a [regexManagers](../../manager/regex/):
 
 Options:
 
 | option                     | default | description                                                                                                                                                              |
 | -------------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| defaultRegistryUrlTemplate | ""      | url used if no `registryUrl` is provided when looking up new releases                                                                                                    |
+| defaultRegistryUrlTemplate | ""      | URL used if no `registryUrl` is provided when looking up new releases                                                                                                    |
 | format                     | "json"  | format used by the API. Available values are: `json`, `plain`                                                                                                            |
 | transformTemplates         | []      | [jsonata rules](https://docs.jsonata.org/simple) to transform the API output. Each rule will be evaluated after another and the result will be used as input to the next |
```

Comment:
I believe the prefered spelling is JSONata

Reply:
You're right! 😄 

```suggestion
| transformTemplates         | []      | [JSONata rules](https://docs.jsonata.org/simple) to transform the API output. Each rule will be evaluated after another and the result will be used as input to the next |
```